net-dns/dnsdist/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>bgo@9dt.de</email>
+		<name>Vladimir Datsevich</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+	</longdescription>
+	<use>
+		<flag name="clang">Use Clang compiler instead of GCC</flag>
+		<flag name="dnscrypt">Enable DNSCRypt support</flag>
+		<flag name="readline">Enable support for libreadline, a GNU line-editing library that almost everyone wants</flag>
```

Comment:
Do not repeat the descriptions that match global flags. This file is used to describe local flags that are not global flags or that are different from global.